source/AMDSClientUi.h

```diff
@@ -36,17 +38,17 @@ private slots:
 	void onNetworkSessionOpened();
 
 	/// slot to handle the signal of newServerConnected (add the serverIdentifier to the combox and update the ui displays -- buffernames and active connections)
-	void onNewServerConnected(QString &serverIdentifier);
+	void onNewServerConnected(QString serverIdentifier);
```

Comment:
I've always been a little bit confused as to whether passing classes via signals/slots should be by value or const. ref. It always seemed a little odd to me that you specify the signatures as const Class&, but then the connect signal removes the ref-ness and you end up with 
```C++
connect(this, SIGNAL(foo(Class)), that, SLOT(onFoo(Class)))
```
and not
```C++
connect(this, SIGNAL(foo(const Class&)), that, SLOT(onFoo(const Class&)))
```

Anyway, I set up a test to pass a class through signals/slots by value and by const lvalue ref.. Indeed each passing of the value incurs one extra copy construction than the const ref. as you'd expect.